.gitignore

```diff
@@ -0,0 +1,3 @@
+/build/
+/source/percona-theme/
+.DS*
```

Comment:
??

Reply:
Because these directories are added to the build. They should not be in the repository.